### ISSUE TITLE
How do you set the MEAN?

### ISSUE THREAD

**Issue by LikaQie** [AlfredXiangWu/face_verification_experiment#19]
Thanks for sharing your project ! I just follow your steps of alignment and extract features using your model and caffe_ftr.py，but I can just get acc 94%  on LFW. So I want to ask whether you set the mean of data. And if you do ,how do you make your test data mean file?

**Comment by AlfredXiangWu** [comment]
I don't use the mean file for training and testing.

**Comment by LikaQie** [comment]
Thanks for your reply! 
I use the Yi sun's detection and alignment method, and about 200 photos fail to detect the faces, do you think it would impact my results? I think the impact should be very small.
And I can not find out what it's wrong. Do you have some advice to me?

**Comment by AlfredXiangWu** [comment]
According to my experiments and others' email, the situation you mentioned may lead to 1%~2% dropped. 

To improve the results on LFW, some advice is as follows：
1. Check the alignment method between training set and testing set. The image size for training set is 144*144, while it is 128*128 for testing. So it is possible to make a mistake.
2. Don't decrease the learning rate too quickly. I fixed the learning rate(1e-3) for about 1,000,000 iterations.
